### PR TITLE
feat(messaging): message channels list

### PR DESCRIPTION
**Pull request by thomas-mauran**
- show all the message channels of the current user

### CODE REVIEW

File: front/src/components/messaging/MessageChannelItem.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          }}
+        />
+        <View>
+          <Text
+            variant='labelLarge'
+            style={styles.nameText}
+          >{`${messageChannel.employer.firstName} ${messageChannel.employer.lastName}`}</Text>
+
+          <Text>{`${messageChannel.lastMessage}`}</Text>
+        </View>
```
Comment (Kuruyia) [inline front/src/components/messaging/MessageChannelItem.tsx:67]: Can you vertically center those two texts?

**Review by Kuruyia — CHANGES_REQUESTED** [review]
Nice work!

File: front/src/components/messaging/MessageChannelItem.tsx (hunk 1 above)
Comment (thomas-mauran) [reply front/src/components/messaging/MessageChannelItem.tsx:67]: image
It seems a bit odd centered no ?
tell me what you think

Comment (thomas-mauran) [reply front/src/components/messaging/MessageChannelItem.tsx:67]: ping @Kuruyia

Comment (Kuruyia) [reply front/src/components/messaging/MessageChannelItem.tsx:67]: Damn! I meant in the other axis, so both texts are centered with the profile picture.

Comment (thomas-mauran) [reply front/src/components/messaging/MessageChannelItem.tsx]: image


done here it what it looks like

Comment (Kuruyia) [reply front/src/components/messaging/MessageChannelItem.tsx]: I think it looks better!

File: front/src/components/messaging/MessageChannelItem.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  },
+  profilePicture: {
+    borderRadius: 48,
+    height: 60,
+    width: 60,
+  },
+  verticalCenter: {
+    marginBottom: 'auto',
+    marginTop: 'auto',
+  },
```
Comment (Kuruyia) [inline front/src/components/messaging/MessageChannelItem.tsx:29]: You can use flexbox properties instead:

```suggestion
  verticalCenter: {
    justifyContent: 'center',
  },
```

**Review by Kuruyia — CHANGES_REQUESTED** [review]

**Pull request merged**